app/styles/toggle-buttons.less

```diff
@@ -0,0 +1,26 @@
+.oss-toggle-buttons-container {
+  .border-radius-default;
+  .background-color-gray-100;
+  .text-color-default;
+
+  display: flex;
+  flex: 1;
+  height: 30px;
+  padding: 3px;
+
+  .oss-toggle-buttons-btn {
+    .border-radius-default;
+
+    display: flex;
+    flex: 1;
+    justify-content: center;
+    align-items: center;
+    padding: 3px;
```

Comment:
```suggestion
  padding: var(--spacing-px-3);
```

Reply:
use variable in https://github.com/upfluence/oss-components/commit/58f8b2e7f8aa651fd5c6e4bcbaf3abf334763996